Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -794,3 +794,9 @@ def _uniqueFirstLayers(layerList):
                               label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
 phase2_timing_layer.toModify( trackValidatorGsfTracks,
                               label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+
+from Configuration.Eras.Modifier_phase2_timing_layer_tile_cff import phase2_timing_layer_tile
+from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toModify( generalTracksFromPV, 
+                              timesTag  = cms.InputTag('tofPID:t0'), 
+                              timeResosTag = cms.InputTag('tofPID:sigmat0') )
```

Comment:
from https://github.com/cms-sw/cmssw/blob/02d4198c0b6615287fd88e9a8ff650aea994412e/CommonTools/RecoAlgos/src/TrackWithVertexSelector.cc#L27 it looks like the cms. types can be dropped

Reply:
Yes, for these two lines it could be dropped.  Do you want me to do it in this PR, or as part of further cleanup later?